### PR TITLE
v1.15 Release Notes

### PR DESCRIPTION
**Pull request by mrpollo**
Final round of changes 

EDIT:

After merging this PR:

- [ ] back porting the commit from this PR to v1.15 branch
- [ ] Merge #3380
- [ ] Merge #3382
- [ ] Merge #3383

These update the front pages of the older releases, and also add v1.15 to their version switcher

### CODE REVIEW

**Comment by mrpollo** [comment]
@bresch I have added your suggestions from Discord please let me know if I captured everything the way you intended.

File: en/releases/1.15.md [hunk 1]
```diff
@@ -155,28 +165,28 @@ Removals:
...
 
 ### Fixed-Wing
 
 - [Simplified airspeed sensor configuration](../config_vtol/vtol_without_airspeed_sensor.md):
-  Replace parameter `CBRK_AIRSPD_CHK` with [SYS_HAS_NUM_ASPD](../advanced_config/parameter_reference.md#SYS_HAS_NUM_ASPD) and renamed parameter `FW_ARSP_MODE` to [FW_USE_AIRSPD](../advanced_config/parameter_reference.md#FW_USE_AIRSPD).
-  To be able to arm without an airspeed sensor set `SYS_HAS_NUM_ASPD` to 0.
-  To not use the airspeed sensor data in the controller, set `FW_USE_AIRSPD` to 0.
-  Added to PX4 in [PX4-Autopilot#22510](https://github.com/PX4/PX4-Autopilot/pull/22510).
-- Improvement: Fixed-wing high (dynamic) wind hardening [PX4-Autopilot#21764](https://github.com/PX4/PX4-Autopilot/pull/21764)
+  - Replacef parameter `CBRK_AIRSPD_CHK` with [SYS_HAS_NUM_ASPD](../advanced_config/parameter_reference.md#SYS_HAS_NUM_ASPD) and renamed parameter `FW_ARSP_MODE` to [FW_USE_AIRSPD](../advanced_config/parameter_reference.md#FW_USE_AIRSPD).
```
Comment (bresch) [inline en/releases/1.15.md:177]:
```suggestion
  - Replace parameter `CBRK_AIRSPD_CHK` with [SYS_HAS_NUM_ASPD](../advanced_config/parameter_reference.md#SYS_HAS_NUM_ASPD) and renamed parameter `FW_ARSP_MODE` to [FW_USE_AIRSPD](../advanced_config/parameter_reference.md#FW_USE_AIRSPD).
```

**Review by bresch — COMMENTED** [review]
All good on the navigation side, thanks @mrpollo

File: en/releases/1.15.md [hunk 2]
```diff
@@ -1,184 +1,199 @@
 # PX4-Autopilot v1.15 Release Notes
```
Comment (hamishwillee) [inline en/releases/1.15.md:1]: @mrpollo I rewrote this section, so it needs to be checked.

**Comment by mrpollo** [comment]
Thanks for the help @hamishwillee I took care of everything including the backport

**Pull request merged**